helpers/api.js

```diff
@@ -1,36 +1,55 @@
 import { AsyncStorage } from 'react-native';
 
 const handleResponse = (response) => {
-    return response.json()
-        .then((json) => {
-            if (!response.ok) {
-                const { message } = json;
-                const error = Object.assign({}, json, {
-                    status: response.status,
-                    message: message
-                });
+    return response.json().then((json) => {
+        if (!response.ok) {
+            const { message } = json;
+            const error = Object.assign({}, json, {
+                status: response.status,
+                message: message
+            });
 
-                return Promise.reject(error);
-            }
-            return json;
-        });
-}
+            return Promise.reject(error);
+        }
+        return json;
+    });
+};
 
 const storeData = ({ access_token, user }) => {
     try {
-        const userData = [
-            ['userToken', access_token],
-            ['user', JSON.stringify(user)]
-        ];
-        AsyncStorage.multiSet(userData);
+        const userData = JSON.stringify({
```

Comment:
i did this change earlier, instead of keeping a separate thing for user and usertoken, i just combine them here before storing so it's more easily accessible.